frontend/src/metabase/home/components/HomeLayout/HomeLayout.tsx

```diff
@@ -1,57 +1,83 @@
-import type { ReactNode } from "react";
-import { useState } from "react";
+import { useDisclosure } from "@mantine/hooks";
+import type { PropsWithChildren } from "react";
 import { t } from "ttag";
 
-import Tooltip from "metabase/core/components/Tooltip/Tooltip";
 import { useSelector } from "metabase/lib/redux";
 import { getUserIsAdmin } from "metabase/selectors/user";
 import { getLandingPageIllustration } from "metabase/selectors/whitelabel";
+import {
+  BackgroundImage,
+  Box,
+  Button,
+  Group,
+  Icon,
+  Space,
+  Text,
+  Tooltip,
+} from "metabase/ui";
 
 import { CustomHomePageModal } from "../CustomHomePageModal";
 import { HomeGreeting } from "../HomeGreeting";
 
-import {
-  LayoutBody,
-  LayoutEditButton,
-  LayoutIllustration,
-  LayoutRoot,
-} from "./HomeLayout.styled";
-
-interface HomeLayoutProps {
-  children?: ReactNode;
-}
-
-export const HomeLayout = ({ children }: HomeLayoutProps): JSX.Element => {
-  const [showModal, setShowModal] = useState(false);
+export const HomeLayout = ({ children }: PropsWithChildren): JSX.Element => {
+  const [isModalOpen, { open: openModal, close: closeModal }] =
+    useDisclosure(false);
   const isAdmin = useSelector(getUserIsAdmin);
   const landingPageIllustration = useSelector(getLandingPageIllustration);
 
+  const containerProps = landingPageIllustration
+    ? {
+        bgsz: landingPageIllustration?.isDefault
+          ? "max(min(1728px, 260vh), 100%) auto"
+          : "100% auto",
+        src: landingPageIllustration.src,
+      }
+    : {};
+
   return (
-    <LayoutRoot data-testid="home-page">
-      {landingPageIllustration && (
-        <LayoutIllustration
-          data-testid="landing-page-illustration"
-          backgroundImageSrc={landingPageIllustration.src}
-          isDefault={landingPageIllustration.isDefault}
-        />
-      )}
+    <Box
+      component={BackgroundImage}
+      w="100%"
+      h="100%"
+      bgr="no-repeat"
+      bgp="bottom"
+      p={{
+        base: "md",
+        md: "3rem 4rem",
+        lg: "4rem 7rem 2rem",
+        xl: "10rem 15rem 4rem",
+      }}
+      data-testid="home-page"
```

Comment:
Since the background image is a part of the root layout here I've just made the `data-testid = home-page`. It seems like `landing-page-illustration` is only used once so I don't think this would be an issue